src/app/components/TeamMemberProfile.svelte

```diff
@@ -49,7 +49,7 @@
     <a href={githubUser.htmlURL} data-cy="contributor-image-{githubUser.login}">
       <img src={githubUser.avatarURL} alt={`${githubUser.name}'s avatar`} />
     </a>
-    <p class="profile-name">{githubUser.name}</p>
+    <p class="profile-name">{githubUser.name || githubUser.login}</p>
```

Comment:
I guess some GitHub users are missing username?

Reply:
Yes! I didn't want to show `null` for past collaborators' names!